docs/usecase/usage-example.md

```diff
@@ -0,0 +1,64 @@
+# Use cases
+
+This document will introduce you to the example of what Vald can do.
+Vald is a highly scalable distributed fast approximate nearest neighbor dense vector search engine, which uses [NGT](https://github.com/yahoojapan/NGT) as the core engine of Vald, and Vald manages to integrate with Kubernetes.
+
+In general, you cannot search your unstructured data using the inverted index, like images and videos.
+By using a vector search engine like Vald, you can search the corresponding unstructured data by converting them to the corresponding feature vectors and process in Vald.
+
+Here are some general use cases of Vald or vector search engine.
+
+1. Image and video processing
```

Comment:
![badge](https://img.shields.io/badge/review-suggestion-blue.svg?style=flat-square)
I think it is better to write the categories as headlines.
```suggestion
## Image and video processing
```